bigtable-client-core/src/main/java/com/google/protobuf/BigtableZeroCopyByteString.java

```diff
@@ -8,10 +8,10 @@
  * of the objects de-serialized from the wire (which already do one copy, on
  * top of the copies the JVM does to go from kernel buffer to C buffer and
  * from C buffer to JVM buffer).
- *
- * @since 0.96.1
  */
 public final class BigtableZeroCopyByteString extends LiteralByteString {
```

Comment:
This class has the basic function of efficient copying of byte[] to and from protobufs.  

Protobufs use ByteString to wrapy byte[]. We need an efficient way to write byte[] into our protobuf objects, and that's where our BigtableZeroCopyByteString comes in.  We can't instantiate a LiteralByteString directly, since by default, protobuf doesn't allow you to create the more efficient LiteralByteString directly; rather, it makes you copy the entire byte[] via ByteString.copyOf() which copies the byte[] and then constructs a new LiteralByteString.

We need new BigtableZeroCopyByteString when writing to protobuf objects and zeroCopyGetBytes when reading from protobuf objects.